### ISSUE TITLE
Plugin crashes with `style`-block

### ISSUE THREAD

**Issue by codekie** [tomblachut/svelte-intellij#125]
Webstorm and OS version:

```
WebStorm 2020.2 EAP
Build #WS-202.5792.46, built on June 18, 2020
Runtime version: 11.0.7+10-b944.12 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.4
```

As soon as I add a `style`-block to a svelte-file, with the most recent Webstorm EAP version ...

eg:
```html
<style>
	p {
		color: purple;
		font-family: 'Comic Sans MS', cursive;
		font-size: 2em;
	}
</style>

<p>Styled!</p>
```

... the plugin crashes with following exception:

```
ava.lang.NoSuchFieldError: ourDefaultStyleLanguage
	at dev.blachut.svelte.lang.parsing.html.SvelteHtmlLexer.access$getOurDefaultStyleLanguage$p$s133750273(SvelteHtmlLexer.kt:9)
	at dev.blachut.svelte.lang.parsing.html.SvelteHtmlLexer.getStyleLanguage(SvelteHtmlLexer.kt:49)
	at com.intellij.lexer.HtmlLexer.getTokenType(HtmlLexer.java:52)
	at com.intellij.lang.impl.TokenSequence$Builder.performLexing(TokenSequence.java:118)
	at com.intellij.lang.impl.TokenSequence.performLexing(TokenSequence.java:68)
	at com.intellij.lang.impl.PsiBuilderImpl.performLexing(PsiBuilderImpl.java:184)
	at com.intellij.lang.impl.PsiBuilderImpl.<init>(PsiBuilderImpl.java:157)
	at com.intellij.lang.impl.PsiBuilderImpl.<init>(PsiBuilderImpl.java:111)
	at com.intellij.lang.impl.PsiBuilderFactoryImpl.createBuilder(PsiBuilderFactoryImpl.java:37)
	at com.intellij.psi.tree.ILazyParseableElementType.doParseContents(ILazyParseableElementType.java:79)
	at com.intellij.psi.tree.IFileElementType.parseContents(IFileElementType.java:53)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.lambda$ensureParsed$0(LazyParseableElement.java:192)
	at com.intellij.psi.impl.DebugUtil.performPsiModification(DebugUtil.java:565)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.ensureParsed(LazyParseableElement.java:191)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:242)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:42)
	at com.intellij.psi.templateLanguages.RangeCollectorImpl.findFirstSuitableElement(RangeCollectorImpl.java:237)
	at com.intellij.psi.templateLanguages.RangeCollectorImpl.insertOuterElementsAndRemoveRanges(RangeCollectorImpl.java:163)
	at com.intellij.psi.templateLanguages.TemplateDataElementType.lambda$parseContents$0(TemplateDataElementType.java:78)
	at com.intellij.psi.impl.DebugUtil.performPsiModification(DebugUtil.java:575)
	at com.intellij.psi.templateLanguages.TemplateDataElementType.parseContents(TemplateDataElementType.java:77)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.lambda$ensureParsed$0(LazyParseableElement.java:192)
	at com.intellij.psi.impl.DebugUtil.performPsiModification(DebugUtil.java:565)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.ensureParsed(LazyParseableElement.java:191)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:242)
	at com.intellij.psi.impl.source.tree.CompositeElement.findLeafElementAt(CompositeElement.java:126)
	at com.intellij.psi.impl.source.tree.CompositeElement.findLeafElementAt(CompositeElement.java:33)
	at com.intellij.psi.AbstractFileViewProvider.findElementAt(AbstractFileViewProvider.java:247)
	at com.intellij.psi.AbstractFileViewProvider.findElementAt(AbstractFileViewProvider.java:217)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.tryOffset(InjectedLanguageUtil.java:475)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageUtil.findInjectedElementNoCommit(InjectedLanguageUtil.java:461)
	at com.intellij.psi.impl.source.tree.injected.InjectedLanguageManagerImpl.findInjectedElementAt(InjectedLanguageManagerImpl.java:283)
	at com.intellij.openapi.editor.EditorMouseHoverPopupManager.createContext(EditorMouseHoverPopupManager.java:374)
	at com.intellij.openapi.editor.EditorMouseHoverPopupManager.lambda$handleMouseMoved$3(EditorMouseHoverPopupManager.java:163)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.insideReadAction(NonBlockingReadActionImpl.java:486)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$attemptComputation$2(NonBlockingReadActionImpl.java:443)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1110)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:77)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:153)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:114)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:56)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:629)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:581)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:56)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:111)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:77)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.attemptComputation(NonBlockingReadActionImpl.java:451)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$transferToBgThread$0(NonBlockingReadActionImpl.java:384)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by codekie** [comment]
Probably will be resolved with #123

**Comment by tomblachut** [comment]
Closing, please reopen if problem persists in https://github.com/tomblachut/svelte-intellij/releases/tag/v0.12.2
